pkg/pillar/cmd/volumemgr/updatestatus.go

```diff
@@ -163,6 +179,9 @@ func doDownload(ctx *volumemgrContext, status *types.VolumeStatus) bool {
 		status.State = ds.State
 		changed = true
 	}
+	if status.DownloadOrigin.IsContainer {
+		status.DownloadOrigin.MaxSizeBytes = ds.Size
```

Comment:
This is not the current size; status.TargetSizeBytes tracks the current size AFAIK

Reply:
We are setting up the download size for containers here because it is 0 in config. We can set ```TargetSizeBytes ``` equal to the download size for the containers.

Reply:
OK